pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -79,12 +79,20 @@ const (
 	// CheckServiceAffinityPred defines the name of predicate checkServiceAffinity.
 	CheckServiceAffinityPred = "CheckServiceAffinity"
 	// MaxEBSVolumeCountPred defines the name of predicate MaxEBSVolumeCount.
+	// DEPRECATED
```

Comment:
Please point out what the replacement is in the comment. Without a clear replacement, deprecating an existing feature causes confusion.

Reply:
But it is mentioned right there!

> // DEPRECATED
	// All cloudprovider specific predicates are deprecated in favour of MaxCSIVolumeCountPred.

`MaxCSIVolumeCountPred` will support all these in-tree volume types via migration path we discussed. I am working on updating the Volume limit KEP as per what we discussed - https://github.com/kubernetes/enhancements/pull/730 .

Reply:
`MaxCSIVolumeCountPred` will support these in-tree volume types even if you are not using CSI.